src/arvtool.c

```diff
@@ -394,6 +399,164 @@ arv_tool_control (int argc, char **argv, ArvDevice *device)
         }
 }
 
+static void
+arv_tool_network (int argc, char **argv, ArvDevice *device)
+{
+        if (!ARV_IS_GV_DEVICE (device))
+                printf ("This is not a GV device\n");
```

Comment:
Don't use an if else structure, but return early if the device is not a GigE one.